src/js/components/Button/__tests__/__snapshots__/Button-test.js.snap

```diff
@@ -1027,6 +1027,12 @@ exports[`Button icon label 1`] = `
   -webkit-font-smoothing: antialiased;
 }
 
+@media only screen and (max-width:768px) {
+  .c3 {
+    width: 6px;
+  }
+}
+
```

Comment:
Is this change implying Button's behavior will change for `xlarge` breakpoint size?

Reply:
See prior comment.